fastestimator/network.py

```diff
@@ -145,8 +146,20 @@ def _collect_models(ops: Iterable[Union[TensorOp, Scheduler[TensorOp]]]) -> Set[
     return models
 
 
+def _associate_loss(ops: Iterable[Union[TensorOp, Scheduler[TensorOp]]]):
```

Comment:
shouldn't this functionality be handled inside the model op?

Reply:
interesting! I think it would be much easier.

Reply:
done, now it's much clearer